test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -155,6 +198,7 @@ private void RestoreProjectOrSolution(string workingDirectory, string fileName,
         /// </summary>
         internal CommandRunnerResult RunDotnet(string workingDirectory, string args, bool ignoreExitCode=false)
         {
+            CreateTempGlobalJson(workingDirectory);
```

Comment:
I don't think it needs added here. 
It should be done at creation time.

Reply:
Thanks! Removed already. Only keep CreateTempGlobalJson in CreateDotnetToolProject  and CreateDotnetNewProject. Will check if manually fix for other tests afterwards.